vyxal/structure.py

```diff
@@ -11,23 +14,23 @@ def transpile(self) -> str:
         return "{}"
 
     def __repr__(self):
-        return str([self.__class__, self.branches])
+        return f"{type(self).__name__}({repr(self.branches)})"
 
 
 class GenericStatement(Structure):
     """
     Elements and so on
     """
 
-    def __init__(self, branches: list[list["Structure"]]):
+    def __init__(self, branches: List[List["Structure"]]):
         super().__init__(branches)
 
     def transpile(self) -> str:
         return super().transpile()
 
 
 class IfStatement(Structure):
-    def __init__(self, branches: list[list[Structure]]):
+    def __init__(self, branches: List[List[Structure]]):
         super().__init__(branches)
         self.truthy = branches[0]
```

Comment:
maybe `branches` should be a variadic argument (`*branches`) so it can be pattern-matched better rather than extracting bits of it one at a time?

Reply:
Good idea, will get on that after fixing more minor things.